api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -442,6 +442,10 @@ type FelixConfigurationSpec struct {
 	// BPFPolicyDebugEnabled when true, Felix records detailed information
 	// about the BPF policy programs, which can be examined with the calico-bpf command-line tool.
 	BPFPolicyDebugEnabled *bool `json:"bpfPolicyDebugEnabled,omitempty"`
+	// BPFForceTrackPacketsFromIfaces forces BPF to track packets from a list of specified interfaces.
+	// Essentially this excludes the specified list of interfaces from being NOTRACKed in IP Tables.
```

Comment:
```suggestion
	// BPFForceTrackPacketsFromIfaces in BPF mode, forces traffic from these interfaces 
	// to skip Calico's iptables NOTRACK rule, allowing traffic from those interfaces to be
	// tracked by Linux conntrack.  Should only be used for interfaces that are not used for
	// the Calico fabric.  For example, a docker bridge device for non-Calico-networked 
	// containers.
```
Suggest rewording to avoid implying that BPF is doing the tracking

Reply:
Also it seems like we allow iptables style prefix like `docker+` so that needs to be documented here, saying, that you can use a specific device like `docker0` or a prefix like `docker+`. Note that we use regexps in most places. It should also state what the default is like `[Default: docker+]`, see the other options for examples.

Reply:
Thanks @tomastigera - I realized that `[Default: docker+]` was omitted - added now and re did `make generate` and confirm visible upon `kubectl explain felixconfig.spec | grep -7 bpfForceTrackPacketsFromIfaces`